app/src/main/java/app/revanced/integrations/youtube/patches/components/DescriptionComponentsFilter.java

```diff
@@ -45,20 +44,42 @@ public DescriptionComponentsFilter() {
                 "transcript_section"
         );
 
+        macroMarkersCarousel = new StringFilterGroup(
+                null,
+                "macro_markers_carousel.eml"
+        );
+
+        chaptersSectionBufferGroup = new ByteArrayFilterGroup(
+                Settings.HIDE_CHAPTERS_SECTION,
+                "chapters_horizontal_shelf"
+        );
+
+        keyConceptsSectionBufferGroup = new ByteArrayFilterGroup(
+                Settings.HIDE_KEY_CONCEPTS_SECTION,
+                "learning_concept_macro_markers_carousel_shelf"
+        );
+
         addPathCallbacks(
                 attributesSection,
-                chaptersSection,
                 infoCardsSection,
                 podcastSection,
-                transcriptSection
+                transcriptSection,
+                macroMarkersCarousel
         );
     }
 
-
     @Override
     boolean isFiltered(@Nullable String identifier, String path, byte[] protobufBufferArray,
                        StringFilterGroup matchedGroup, FilterContentType contentType, int contentIndex) {
-        if (exceptions.matches(path)) return false;
```

Comment:
Why were the exceptions removed?

Reply:
Removed by accident. Added it back in b9656614d1b51e20b9a30928e139577b11f9766d